src/ngraph/autodiff/adjoints.cpp

```diff
@@ -106,48 +140,60 @@ autodiff::Adjoints::Adjoints(const std::shared_ptr<Node>& y, const std::shared_p
                 nodes_to_check.push_front(arg);
             }
         }
-        node->generate_adjoints(*this, get(node));
+        auto deltas = get(node);
+        if (node->get_outputs().size() > 1)
+        {
+            node->generate_adjoints(*this, deltas);
+        }
+        else
```

Comment:
Better make sure there is at least one output before doing this. We don't have any 0-output nodes, but who knows what might happen in the future.